plugins/search-indexer/.gitignore

```diff
@@ -0,0 +1,10 @@
+# Don't check auto-generated stuff into git
+coverage
+node_modules
+stats.json
+package-lock.json
```

Comment:
Is there a reason why this is being ignored as usually it should be commited?

Reply:
Good point 🤔 , but this is how Strapi constructed the plugin when I created it with the Strapi CLI tool, so there's probably a reason for it.